helm/defectdojo/templates/extra-fixtures.yaml

```diff
@@ -0,0 +1,28 @@
+{{- $fullName := include "defectdojo.fullname" . -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $fullName }}-extra-fixtures
+  labels:
+    app.kubernetes.io/name: {{ include "defectdojo.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ include "defectdojo.chart" . }}
+data:
+  {{- range $i, $fixture := .Values.extraFixtures }}
+  {{- printf "extra_%03d.json" $i | nindent 2 }}: |
+    {{- $models := dict }}
+    {{- $output := list }}
+    {{- range $j, $data := $fixture.data }}
+    {{- if empty $data.model }}
+    {{- printf "'model' key missing from extraFixture %d, data item %d" $i $j | fail }}
```

Comment:
I decided it was important to fail helm installations if extra fixtures are missing the required fields. Is this important?